src/ConcreteRArray.jl

```diff
@@ -201,7 +201,7 @@ function Base.show(io::IO, X::ConcreteRArray)
         println(io, "<Empty buffer>")
         return nothing
     end
-    str = sprint(show, convert(Array, X))
+    str = sprint(show, convert(Array, X); context=IOContext(io))
     return print(io, "$(typeof(X))($(str))")
```

Comment:
we can prevent allocating the str with something like:
```suggestion
    print(io, "$(typeof(X))(")
    show(io, convert(Array, X))
    print(io, ")")
    return nothing
```

Reply:
done